hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenManagementSpecs.java

```diff
@@ -575,14 +575,15 @@ public HapiSpec updateIdVariantsTreatedAsExpected() {
                         cryptoCreate("autoRenewAccount"),
                         tokenCreate("t").adminKey("adminKey"))
                 .when()
-                .then(submitModified(
-                        withSuccessivelyVariedBodyIds(),
-                        () -> tokenUpdate("t").autoRenewPeriod(7776000L).autoRenewAccount("autoRenewAccount")));
+                .then(submitModified(withSuccessivelyVariedBodyIds(), () -> tokenUpdate("t")
+                        .autoRenewPeriod(7776000L)
+                        .autoRenewAccount("autoRenewAccount")
+                        .signedBy(DEFAULT_PAYER, "adminKey", "autoRenewAccount")));
     }
 
     @HapiTest
     public HapiSpec wipeIdVariantsTreatedAsExpected() {
-        return defaultHapiSpec("updateIdVariantsTreatedAsExpected")
+        return defaultHapiSpec("wipeIdVariantsTreatedAsExpected")
```

Comment:
We should find an intern to write a script to grovel all our HAPI tests for this mismatch.  How many times have I see this copy/paste error?